### PR TITLE
UI Improvements for hotkeys

### PR DESCRIPTION
**Pull request by paolodamico**
- Introduces a tooltip with information on hotkeys.

- Introduces peer dependency to AntD to show tooltips.
    - For linting and microbundle to work properly, Ant had to be added as a dev dependency as well. There should be a better approach for this.
﻿

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -155,6 +160,17 @@ export const Player = forwardRef<PlayerRef, Props>(function Player(
             seek(currentTime + JUMP_TIME_MS / 2)
         } else if (event.key === 'f') {
             toggleFullScreen()
+        } else if (event.key === 'a') {
+            props.onPrevious && props.onPrevious()
+        } else if (event.key === 'd') {
+            props.onNext && props.onNext()
+        } else {
+            // Playback speeds shortcuts
+            for (let i = 0; i < PLAYBACK_SPEEDS.length; i++) {
```
Comment (macobo) [inline src/index.tsx:169]: In youtube 0-9 jump through the video (0 - start, 1 - 1/10th of the video, etc)

File: src/index.tsx [hunk 2]
```diff
@@ -239,49 +255,82 @@ export const Player = forwardRef<PlayerRef, Props>(function Player(
...
+                                ) : (
+                                    <IconPlay
+                                        onClick={togglePlayPause}
+                                        className='ph-rrweb-controller-icon ph-rrweb-controller-icon-play-pause'
+                                    />
+                                )}
+                            </span>
+                        </Tooltip>
+                        <Tooltip
+                            title={`Back ${JUMP_TIME_MS} secs (← left arrow)`}
```
Comment (macobo) [inline src/index.tsx:274]: This is incorrect - it's JUMP_TIME_MS milliseconds ;)

File: package.json [hunk 3]
```diff
@@ -39,6 +40,7 @@
     "@types/react-transition-group": "^4.4.0",
     "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
+    "antd": "^4.1.1",
```
Comment (macobo) [inline package.json:43]: This is a really heavy dependency to be adding here - it adds more than 50% to the bundle.

I'll see whether I can get rid of it.

Comment (paolodamico) [reply package.json]: In the context of this package I totally agree that using `rc-tooltip` is better than adding the full `antd` package, however viewing it from overall PostHog perspective, we're actually introducing an additional dependency (that also doesn't have the exact same visuals that we already have in the rest of the app). What I think would be the best approach here is to use AntD's tooltip (if the package is available in the target project), and if not use the default browser's tooltip mechanism. Thoughts?

Comment (paolodamico) [reply package.json]: Relevant for discussion, https://github.com/PostHog/posthog/issues/2556

**Pull request merged**